### PR TITLE
Bump minimum gtk+ version to 3.22.26

### PR DESCRIPTION
**Pull request by jplatte**
@hellcp Is this fine with you? All other recent contributors indicated that it's fine with them (I would put a link to the relevant chat bits here, but don't know how)

### CODE REVIEW

**Comment by hellcp** [comment]
GTK+3 seems to have stagnated on 3.22 so it's really fine, majority of platforms support it :smile:

File: src/app/launch.rs [hunk 1]
```diff
@@ -47,9 +47,10 @@ pub(super) fn connect(
         // Parts mentioning shortcuts here can be then removed
 
         act_shortcuts.connect_activate(clone!(app => move |_, _| {
-            let dialog: gtk::Window = gtk::Builder::new_from_resource("/org/fest-im/fest/gtk/help-overlay-old.ui")
-                .get_object("help_overlay_old")
-                .expect("Couldn't find help_overlay_old in ui file.");
+            let dialog: gtk::ShortcutsWindow =
+                gtk::Builder::new_from_resource("/org/fest-im/fest/gtk/help-overlay.ui")
```
Comment (hellcp) [inline src/app/launch.rs]: We could just use native way of showing this, with menus, as stated in comment above this line ;)

File: res/gtk/menus.ui [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interface>
-  <requires lib="gtk+" version="3.16"/>
   <menu id="app-menu">
```
Comment (hellcp) [inline res/gtk/menus.ui:3]: This file requires just replacing `app.shortcuts` with `win.show-help-overlay`

**Review by hellcp — CHANGES_REQUESTED** [review]

File: res/gtk/menus.ui (hunk 2 above)
Comment (hellcp) [reply res/gtk/menus.ui:3]: There is no point of adding keyboard shortcuts stuff to launch.rs

**Comment by jplatte** [comment]
@hellcp Thanks for the fast response. Fixed.

**Review by hellcp — APPROVED** [review]
Perfect :D

**Comment by hellcp** [comment]
We won't please Travis with anything, I will prepare Travis setup with docker and newer version of Ubuntu or some other OS later. For now: those changes look alright :D

**Comment by jplatte** [comment]
I manually rebased + pushed this to master. Travis is gone now.

**Pull request closed without merging**